(DO NOT Merge) - Work pending please do not Merge (#9154)
MicrosoftDocs/office-docs-powershell: skype/skype-ps/skype/Enable-CsOnlineDialInConferencingUser.md

> If true, specified toll-free number is used in meeting invitations.
> If false, specified toll-free number is not allowed to be used in meeting invitations.
> The default is true.
> This setting can ONLY be managed using the TeamsAudioConferencingPolicy.

Reviewer: ```suggestion
This setting can ONLY be managed using the TeamsAudioConferencingPolicy cmdlets.
```
I understand that this should be enough.

What about ServiceNumber and TollFreeServiceNumber parameters? I think the same sentence apply for them.

Reviewer 2: The original version is correct as the setting can also be managed from Teams Admin Center through changing the policy, so not only by using cmdlets. The intention was to highlight that the old way of updating AllowTollFreeDialin setting (through Set-CsOnlineDialinConferenincgUser cmdlet) is no longer valid.

Reviewer: Ok, thanks. I understand that we can still use ServiceNumber and TollFreeServiceNumber in Set-CsOnlineDialinConferenincgUser cmdlet.
